code/modules/mining/machine_processing.dm

```diff
@@ -5,6 +5,28 @@
 /obj/machinery/mineral
 	var/input_dir = NORTH
 	var/output_dir = SOUTH
+	var/turf/input_turf = null // The turf that the machines will be listening to for ores/items
+	var/needs_item_input = FALSE // whether or not the machine needs to pick up items, i.e. the ORM picking up ores
+	processing_flags = START_PROCESSING_MANUALLY
+	subsystem_type = /datum/controller/subsystem/processing/fastprocess

```

Comment:
Put the redefined variables at the t op of the object definition, before the object variables, please.

Reply:
Sure. also yeah I'll write some documentation